alexthompson207/rancid-tomatillos
# Feature/testing part 1

## Is this a fix or a feature?

This is a feature, testing all movies view(homepage) UI.

## What is the change?

Installed Cypress to test homepage(all movies view UI)

## What does it fix?

N/A.

## Where should the reviewer start?

Pull down this branch, run npm i, then npm open:cypress. Look at the `all-movies-view.js` file for happy and sad paths.

## How should this be tested?
Pull down this branch, run `npm i`, then `npm open:cypress` all tests should pass.
